pages/programs/baton_roots.html

```diff
@@ -1 +1,121 @@
+</head>
+
+<body>
+
+  <body>
+	<div id="headerBR">
+		<div class="grid-st">
+			<img src="http://www.thewallsproject.org/img/shared/walls_logo.svg"  />
+		</div>
+		<div class="grid-end">
+			<div id="navBR">
+				<div class="navBR-1"><a href="#">Progress<span class="u-BR"></span></a></div>
+				<div class="navBR-2"><a href="#">About<span class="u-BR"></span></a></div>
+				<div class="navBR-3"><a href="#">Contribute<span class="u-BR"></span></a></div>
+				<div class="navBR-4"><a href="#">Contact Us<span class="u-BR"></span></a></div>
+			</div>
+		</div>
+	</div>
+
+
+	<div id="contentBR">
+		<div class="fixedbg"></div>
+		<div id="subcontentBR">
+			<p class="introduction-p">By blending access to fresh food, skill-development, socialization, and art into one location, Capital Fresh brings an
+               opportunity for our community to learn best practices in sustainable agriculture on an urban farm. Developed by The
+               Walls Project, this multi-generational program allows the mergence of hands-on gardening with an artistic
+               environment to ground and inspire community health and wellness.
+
+<br/><br/>
+Student-managed farm areas will allow elementary, middle, and high school students to learn the skills needed to
+               grow and maintain a working business. From planting and harvesting to monitoring quality control, to distribution,
+               students are required to problem solve, anticipate change, manage a workload, and communicate in teams.
+<br/><br/>
+In addition to a student focus, Capital Fresh incorporates an intergenerational garden that will play a larger role in
+               impacting all community members. Improving community connections and reducing social isolation of adults and
+               seniors is a focus of the community garden in addition to growing food. Students are encouraged to work with and
+               get to know their elders. By establishing intergenerational relationships, students gain unspoken societal
+               accountability while elders are able to impart their stories and gain relationships all in the name of a healthy meal.
+ </p>
+		</div>
+
+
+
+
+
+
+<div class="br-root-spacing_intro">
+		<div class="br-root-picture"> </div>
+
+	<div class="br-root-intro">
+	  <h10> INTER-GENERATIONAL<br> FARMING</h10>
+	</div>
+	  <div class="br-root-paragraph">
+		<h6>Connecting all generations through growing fruits, vegetables,<br> herbs, and flowers is the goal of the Intergenerational Community<br> Garden. This raised bed garden, led by Master Gardener volunteers,<br> will reduce social isolation and improve connections to community,<br> while growing a bounty of fresh and healthy food.
+	  </div>
+</div>
+
+	<div class="br-root-cards">
+			<a href="#0" class="br-root-card">
+				<span class="br-root-card-date">12</span>
+				<span class="br-root-card-month">Feb</span>
+				<h3 class="br-root-card-title">Event Title</h3>
```

Comment:
We can't have a block level element `<h3>` inside an inline element, `<a>`. We'll need to make that `<h3>` a `<span>`